app/models/manageiq/providers/azure_stack/inventory/persister/definitions/cloud_collections.rb

```diff
@@ -22,10 +22,4 @@ def add_resource_groups
       builder.add_default_values(:ems_id => manager.id)
     end
   end
-
-  def add_orchestration_stacks
-    add_collection(cloud, :orchestration_stacks) do |builder|
-      builder.add_properties(:model_class => ::ManageIQ::Providers::AzureStack::CloudManager::OrchestrationStack)
```

Comment:
@agrare now that I've added the
```
require_nested OrchestrationStack
```
on the manager this is being picked up correctly so I was able to simplify the persister.

Reply:
:+1: perfect